applications/Liberland_judiciary.md

```diff
@@ -0,0 +1,542 @@
+[Blockchain website](https://liberland.org/blockchain)
+[Project website](https://liberland.org)
+[Mainnet explorer](https://polkadot.js.org/apps/?rpc=wss%253A%252F%252Fmainnet.liberland.org#/)
+[Mainnet frontend](https://blockchain.liberland.org/signin)
+[Testnet explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftestchain.liberland.org%2F#/explorer)
+[Testnet frontend](https://testnet.liberland.org)
+[Metaverse](https://liberverse.net/)
+[Liberland Marketplace](https://market.ll.land/)
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  4 FTE
+- **Total Costs:** 30 000 USD.
+
+### Milestone 1 Simple hybrid contracts
+
+- **Estimated duration:** 1 month
+- **FTE:**  1,5
+- **Costs:** 15,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | MIT |
+| **0b.** | Documentation |  **Inline documentation** of the code and a **tutorial** to spin up a Liberland nodes and test how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Unit tests and testing guide. |
+| **0d.** | Docker | Dockerfile(s) that can be used to test the functionality in this milestone. |
+| 0e. | Article | Publish an **article**/workshop that explains the functioning of the judiciary and its impact on the Polkadot environment |
+| 1. | Substrate: Simple judiciary | Traditional plaintext contracts on chain that can be signed by parties and registered judges |
+| 2. | Substrate: Hybrid contracts | Expanding smart contract solutions to support plaintext clauses, courts as oracles, work with other pallets such as LLM |
+| 3. | Substrate: Default contracts | Develop default contracts for common business, rental and insurance uses |
+| 4. | Frontend| We will deliver a basic frontend to operate the new substrate modules |
+
+
+### Milestone 2 Contract builder and judiciary
+
+- **Estimated Duration:** 1-3 month
+- **FTE:**  1,5
+- **Costs:** 15,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | MIT |
+| **0b.** | Documentation | **Inline documentation** of the code and a **tutorial** to spin up a Liberland nodes and test how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Unit tests and testing guide. |
+| **0d.** | Docker | Dockerfile(s) that can be used to test the functionality in this milestone.
+| 1. | Contract insurace |  Develop default insurance contracts that can be used as collateral provider in other contracts |
+| 2. | Substrate module: Judiciary Market | Module that gathers on-chain data to facilitate the operation of the marketplace of judges to be used as judges and jurisdictions in contracts|
+| 3. | Contract builder | Easy to use frontend interface to build hybrid contracts by adding pre-built smart and traditional clauses, define jurisdictions, handle collateral, handle dispute mode etc. |
```

Comment:
Have you done any work on this yet? Usually, we ask for wireframes or designs at the time of application.